pkg/hds/tracker/healthcheck_generator.go

```diff
@@ -137,7 +137,7 @@ func (g *SnapshotGenerator) envoyHealthCheck(port uint32) *envoy_service_health.
 				Address: &envoy_core.Address{
 					Address: &envoy_core.Address_SocketAddress{
 						SocketAddress: &envoy_core.SocketAddress{
-							Address: "127.0.0.1",
```

Comment:
I mean to use https://www.envoyproxy.io/docs/envoy/v1.25.3/api-v3/service/health/v3/hds.proto.html#service-health-v3-clusterhealthcheck `upstream_bind_config `, we still want to do request to `127.0.0.1` but we want to override source address os iptables can handle it. https://github.com/kumahq/kuma/blob/master/pkg/xds/generator/inbound_proxy_generator.go#L43 like this so we don't do it for no TP deployment

Reply:
Right, fixed